app/controllers/container_topology_controller.rb

```diff
@@ -1,13 +1,8 @@
-class ContainerTopologyController < ApplicationController
-  include TopologyMixin
+class ContainerTopologyController < TopologyController
   before_action :check_privileges
   before_action :get_session_data
   after_action :cleanup_action
   after_action :set_session_data
 
-  private
-
-  def get_session_data
-    @layout = "container_topology"
-  end
+  @layout = "container_topology"
```

Comment:
Yes, that is part of the anomaly. Container topology is different to the othe topologies.